lib/pebbles/pebblifier.flow

```diff
@@ -0,0 +1,387 @@
+import pebbles/pebble_parameters;
+
+export {
+	// It will add to current pebble parameters iScriptBehaviours ("value", "content", "selected") of interactive
+	// material elements rendered on the screen. To make it works all material elements that should be reflected
+	// in pebble should have IScriptId(id) style. These IScriptId.ids must be unique except cases where group of
+	// elements share the same DynamicBehaviour (e.g. MRadios).
+	// PebbleParameter will be &toLowerCase(IScriptId.id) + "_value/content/selected"=value
+	// In case of buttons we can reflect their action only if that button open dialog, that have to be closed
+	// by another one. To make it works you have to pass for openButton style IScriptId(id + "_open")
+	// and for closeButton style IScriptId(id + "_close").
+	MPebblifier(
+		m : Material,
+		controller : PebbleController,
+		ids : [string]
+	) -> Material;
+}
+
+pebblifyMSlider(mf : MaterialFocus) -> Maybe<Pair<PebbleParameterLink, [() -> () -> void]>> {
+	if (toLowerCase(mf.name) == "mslider") {
+		minValue = getIScriptPairValueB(mf.behaviours, "minValue", make(0.0));
+		maxValue = getIScriptPairValueB(mf.behaviours, "maxValue", make(1.0));
+		downAnywhere = getIScriptPairValueB(mf.behaviours, "downAnywhere", make(false));
+		valueB = getIScriptPairValueB(mf.behaviours, "value", cloneBehaviour(minValue));
```

Comment:
this and all similar code will have to be supported very carefully. if something changes in the structure of these material elements, this should be changed accordingly